src/app/server/app.go

```diff
@@ -60,51 +68,68 @@ func NewApp(opts ...AppOptions) *App {
 		),
 	}
 
-	// Define routes and middlewares
-	app.Engine.StaticFS("/static", &assetfs.AssetFS{
-		Asset:    Asset,
-		AssetDir: AssetDir,
-		Prefix:   "static",
-	})
+	// Define static dir
+	app.Engine.Static("/static", "/client")
```

Comment:
It will not work correctly. This project uses approach to serve static files from `go-bindata`. And it won't work because client folder has no compiled app and css bundle. It only has source code of the app. Is I see it nned to write middleware the serve files from bindata